vega/base.py

```diff
@@ -25,31 +24,24 @@ def __init__(self, spec, data=None, opt=None):
     def _prepare_spec(self, spec, data):
         return spec
 
-    def _generate_html(self, id):
-        template = utils.get_content(self.HTML_TEMPLATE)
-        return template.format(id=id)
-
     def _generate_js(self, id, **kwds):
         template = utils.get_content(self.JS_TEMPLATE)
         selector = '#{0}'.format(id)
         payload = template.format(
             selector=selector,
+            id=json.dumps(str(id)),
```

Comment:
Why add `json.dumps`?

Reply:
Okay. Makes sense. Btw, if you want to discuss things more directly, feel free to join https://bit.ly/join-vega-slack.

Reply:
Thanks for link tot he slack channel. I'll give it a go tomorrow.

Reply:
I really don't think this is necessary here, though. The id comes from https://tools.ietf.org/html/rfc4122.html.